extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -510,7 +545,7 @@ export async function createResourceGroup(account: AzureAccount, subscription: a
 export async function getStorageAccountAccessKey(account: AzureAccount, subscription: azureResource.AzureResourceSubscription, storageAccount: azureResource.AzureGraphResource, ignoreErrors: boolean): Promise<GetStorageAccountAccessKeyResult> {
 	const path = `/subscriptions/${subscription.id}/resourceGroups/${storageAccount.resourceGroup}/providers/Microsoft.Storage/storageAccounts/${storageAccount.name}/listKeys?api-version=2019-06-01`;
 	const host = getProviderMetadataForAccount(account).settings.armResource.endpoint;
-	const response = await makeHttpRequest(account, subscription, path, HttpRequestMethod.POST, undefined, ignoreErrors, host);
+	const response = await makeHttpRequest<StorageAccountListKeysResult>(account, subscription, path, HttpRequestMethod.POST, undefined, ignoreErrors, host);
 	return {
 		keyName1: response?.response?.data?.keys[0].value ?? '',
 		keyName2: response?.response?.data?.keys[0].value ?? '',
```

Comment:
@aasimkhan30 This doesn't seem correct...why are we returning the same key value twice? I vaguely remember bringing this up with you but I don't see anything in https://github.com/microsoft/azuredatastudio/pull/16103 so maybe I just imagined it...

I'm also not sure why you named these `keyName` when it's returning the value (keyName is a different property that's returned). Do you recall anything about that?